tests.py

```diff
@@ -111,5 +120,31 @@ def test_json(self):
             print("Issues with (%s) -- %s" % (id, name))
 
 
+class ValidationTest(TestCase):
+    """Validation makes sure the database conforms to its JSON Schema."""
+
+    try:
+        courts = load_courts_db()
+    except:
+        print("JSON FAIL")
+        pass
+
+    def test_validates(self):
+        with open_courts() as f:
+            data = f.read()
+            instance = json.loads(data)
+            with open_schema() as schema_f:
+                schema_data = schema_f.read()
+                schema = json.loads(schema_data)
+
+                try:
+                    jsonschema.validate(
+                        instance=instance,
+                        schema=schema,
+                    )
+                except jsonschema.ValidationError as e:
+                    self.fail("JSON failed validation against schema")
```

Comment:
It seems like this validation would fail without a meaningful message. Is there a better way to do this that gives us something more about how the validation failed?

Reply:
Yes, I'm pretty sure that's possible. `jsonschema.validate()` typically barfs more useful stuff when it raises its exceptions. I'll look into it.

Reply:
Should be fixed, let me know what it does for you.